.env.example

```
@@ -37,4 +37,9 @@ NEXT_PUBLIC_EFP_LIST_RECORDS_MAINNET=0x5289fE5daBC021D02FDDf23d4a4DF96F4E0F17EF
 
 NEXT_PUBLIC_MAINNET_LIMIT=500
 NEXT_PUBLIC_OPTIMISM_LIMIT=500
-NEXT_PUBLIC_BASE_LIMIT=1000
\ No newline at end of file
+NEXT_PUBLIC_BASE_LIMIT=1000
+
+# PostHog product analytics. Leave NEXT_PUBLIC_POSTHOG_KEY empty to disable.
+# Hedge is our managed PostHog proxy; posthog-js sends events to it directly.
+NEXT_PUBLIC_POSTHOG_KEY="phc_wXHXwgojPaFZw3RgzJV3v9TJBFaJZ2L6SjPqwivRZ4wD"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)